persistence.js

```diff
@@ -32,15 +41,14 @@ function MongoPersistence (opts) {
 
 util.inherits(MongoPersistence, CachedPersistence)
 
-MongoPersistence.prototype._connect = function (cb) {
+MongoPersistence.prototype._connect = async function (cb) {
   if (this._opts.db) {
-    cb(null, this._opts.db)
+    await cb(null, this._opts.db)
```

Comment:
Please remove async/await.

Reply:
Can I use promises instead?

Reply:
Unfortunately no as most of the other aedes codes does not use them and it would create a weird error model.